exporters/ibmmq/exporter.yml

```diff
@@ -1,7 +1,7 @@
 # name of the exporter, should match with the folder name
 name: ibmmq
 # version of the package created
-version: 0.4.2
+version: 0.4.3
```

Comment:
if this integration is using the default template I would modify it in this PR

Reply:
It is using the default, but it is already modified in #164 which needs to be merged before this PR. I left this as a draft and set [NEWRELIC-4959-support-scrape-timeout](https://github.com/newrelic/newrelic-prometheus-exporters-packages/tree/NEWRELIC-4959-support-scrape-timeout) branch as base to do so.